### ISSUE TITLE
mirage configure wants already installed package

### ISSUE THREAD

**Issue by sgrove** [mirage/mirage#528]
```
$ env FS=crunch mirage configure --xen --dhcp=true --no-argv
# Detecting depexts using flags: x86_64 linux debian
# The following system packages are needed:
#  - debianutils
#  - libgdbm-dev
#  - libgmp-dev
#  - libpcre3-dev
#  - libsqlite3-dev
#  - libssl-dev
#  - libxen-dev
#  - m4
#  - ncurses-dev
#  - perl
#  - pkg-config
#  - time
#  - zlib1g-dev
# The following new OS packages need to be installed: ncurses-dev
The following command needs to be run through "sudo":
    apt-get install ncurses-dev
Allow ? [Y/n]
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'libncurses5-dev' instead of 'ncurses-dev'
libncurses5-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
# OS packages installation successful
```

1. Perhaps the name of the ncurses package is a bit off? I'm using the `unikernel / mirage` docker image for this
2. Is there a flag to always allow, so it's scriptable?

**Comment by cpitclaudel** [comment]
Same here; mirage reinstalls ncurses-dev every time

**Comment by samoht** [comment]
which version of `depext` do you have?

**Comment by cpitclaudel** [comment]
```
$ opam info depext
             package: depext
             version: 0.9.1
          repository: default
        upstream-url: https://github.com/ocaml/opam-depext/archive/v0.9.1.tar.gz
       upstream-kind: http
   upstream-checksum: dd3d5c03ca7953c16f00d7e6727756fa
            homepage: https://github.com/ocaml/opam-depext
         bug-reports: https://github.com/ocaml/opam-depext/issues
            dev-repo: https://github.com/ocaml/opam-depext.git
              author: Louis Gesbert <louis.gesbert@ocamlpro.com>, Anil Madhavapeddy <anil@recoil.org>
             license: LGPL-3.0 with OCaml linking exception
                tags: flags:plugin
             depends: cmdliner
   installed-version: 0.9.1 [4.03.0]
  available-versions: 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.8.1, 0.9.0, 0.9.1
         description: Query and install external dependencies of OPAM packages

opam-depext is a simple program intended to facilitate the interaction between
OPAM packages and the host package management system. It can perform OS and
distribution detection, query OPAM for the right external dependencies on a set
of packages, and call the OS's package manager in the appropriate way to install
them.
```

**Comment by yomimono** [comment]
I think this was solved by https://github.com/mirage/functoria/pull/66 (which made it into functoria 1.1.1).  I'm closing, but please feel free to reopen if you're still getting annoying behavior around this.
